rootfs/api/serializers.py

```diff
@@ -337,6 +338,15 @@ def validate_healthcheck(self, data):
                 raise serializers.ValidationError(
                     "could not validate {}: {}".format(value, e.message))
 
+        # http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_probe
```

Comment:
i think you can validate this using jsonschema itself
"successThreshold": { "enum": [ 1 ] },